src/build/run_action.mjs

```diff
@@ -93,9 +93,10 @@ export async function runAction(actionPath, ...parameters) {
   try {
     await spawnStream(runner, [...subCommands, resolvedPath, ...parameters]);
   } catch (error) {
-    if (error?.message) {
-      console.error(chalk.red(error.message));
+    if (error?.exitCode) {
+      console.error(chalk.red(`Exit code: ${error.exitCode} ${error.constructor?.name}`));
     }
+    console.error(chalk.red(String(error)));
```

Comment:
I was getting back a `ChildProcess` object in the error variable, which is not really desirable, but that's what happened. So I handle it by looking for the `exitCode` property.

Secondly, instead of logging `error.message` I log `String(error)` which should produce almost identical output for errors (it tends to have an extra `"Error: "` on the front). The benefit of this is, when a non-error is caught here, at least _something_ will be printed!